person.rb

```diff
@@ -6,9 +6,10 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
```

Comment:
- [ ] please kindly ensure there's a method for adding rentals here as well. Here, The person itself should be sent as a parameter to create the new rental, achieving the 'has-many' association`
something like
```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
Fixed.